src/core/router/router.blueprint.tsx

```diff
@@ -2,6 +2,8 @@ import { Navigate, RouteObject, createBrowserRouter, useRouteError } from 'react
 import Layout from 'src/modules/Layout';
 import { FallBack } from 'src/pages/fallback';
 
+import { getSchemasAdaptor } from '../adaptors';
+
```

Comment:
Please use absolute path

Reply:
It's better to use `Absolute path` for two more level directories
Otherwise we should change a lot of places using `./` and `../`